doc/source/index.rst

```diff
@@ -60,3 +60,34 @@ Index
    msg
    run_engine
    api_changes
+
+.. toctree::
+   :hidden:
+   :caption: Data Collection
+
+   bluesky <https://nsls-ii.github.io/bluesky>
+   ophyd <https://nsls-ii.github.io/ophyd>
+
+.. toctree::
+   :hidden:
+   :caption: Data Access and Management
+
+   databroker <https://nsls-ii.github.io/databroker>
+   amostra <https://nsls-ii.github.io/amostra>
+   datamuxer <https://nsls-ii.github.io/datamuxer>
+   suitcase <https://nsls-ii.github.io/suitcase>
+
+.. toctree::
+   :hidden:
+   :caption: Data Analysis
+
+   scikit-beam <http://scikit-beam.github.io/scikit-beam>
+
+.. toctree::
+   :hidden:
+   :caption: GitHub Links
+
+   NSLS-II Repositories <https://github.com/NSLS-II/>
+   Wish List <https://github.com/NSLS-II/wishlist/issues>
```

Comment:
This wishlist repo is basically not used. Before we copy the link across N repos: do we still _want_ to use it?

Reply:
The wish list is most useful from the main page (http://nsls-ii.github.io/).  The content that appears across all *N* repos is most useful from mobile clients where editing the address URL can be cumbersome.

Reply:
Agree with that too. I'll remove the wish list link from the project-specific docs, leaving it at the main page.